src/components/notifications/SMSC.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="mb-3">
+        <label for="smsc-login" class="form-label">{{ $t("API Username") }}</label>
+        <i18n-t tag="div" class="form-text" keypath="wayToGetClickSendSMSToken">
+            <a href="https://smsc.kz/" target="_blank">{{ $t("here") }}</a>
+        </i18n-t>
+        <input id="smsc-login" v-model="$parent.notification.smscLogin" type="text" class="form-control" required>
+        <label for="smsc-key" class="form-label">{{ $t("API Key") }}</label>
+        <HiddenInput id="smsc-key" v-model="$parent.notification.smscPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <div class="form-text">
+            {{ $t("checkPrice", ['СМСЦ']) }}
+            <a href="https://smsc.kz/tariffs/" target="_blank">https://smsc.kz/tariffs/</a>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="smsc-to-number" class="form-label">{{ $t("Recipient Number") }}</label>
+        <input id="smsc-to-number" v-model="$parent.notification.smscToNumber" type="text" minlength="8" maxlength="14" class="form-control" required>
```

Comment:
Is the min/max lenght restricted like this by the Provider?

```suggestion
        <input id="smsc-to-number" v-model="$parent.notification.smscToNumber" type="text" minlength="4" maxlength="15" class="form-control" required>
```
8 might not be the minimum for phone numbers: https://stackoverflow.com/questions/14894899/what-is-the-minimum-length-of-a-valid-international-phone-number

Also 15 are allowed by https://en.wikipedia.org/wiki/Telephone_numbering_plan

Reply:
I checked my number. Its length was its 12 characters.

Reply:
+7XXXXXXXXXX or 8XXXXXXXXXX minlength="11" and maxlength unlimited. Since you can enter multiple numbers